src/Equinor.ProCoSys.IPO.Infrastructure/Repositories/PersonRepository.cs

```diff
@@ -17,5 +17,8 @@ public Task<Person> GetWithSavedFiltersByOidAsync(Guid oid)
             => DefaultQuery
                 .Include(p => p.SavedFilters)
                 .SingleOrDefaultAsync(p => p.Oid == oid);
+
+        public void RemoveSavedFilter(SavedFilter savedFilter)
```

Comment:
This class has no tests

Reply:
True. Adding now. Guess I should make a TECH to add tests in preservation as well? (no PersonRepositoryTests.cs in preservation)